.chloggen/examine-subcommand.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: service
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: A new subcommand to dump the initial configuration after resolving/merging. This is currently in experimental mode.
```

```suggestion
note: Create a new subcommand to dump the initial configuration after resolving/merging.
```
We can call out that the subcommand is experimental below and keep the note a bit shorter.